alerter.c

```diff
@@ -2,19 +2,24 @@
 #include <assert.h>
 
 int alertFailureCount = 0;
+static int FailureCount = 0;
+static int NotFailureCount = 0;
 
-int networkAlertStub(float celcius) {
+int networkAlertStub(float celcius) 

```

Comment:
stub and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.